core/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -282,7 +282,7 @@ describe("IModelTransformer", () => {
     targetDb.close();
   });
 
-  it.skip("should clone Model within same iModel", async () => {
+  it("should clone Model within same iModel", async () => {
```

Comment:
@MichaelBelousov this is the other one you think can be safely deleted?
```
  1) IModelTransformer
       should clone Model within same iModel:
     Error: error inserting
      at IModelImporter.onInsertElement (src\IModelImporter.ts:121:57)
      at IModelImporter.importElement (src\IModelImporter.ts:117:73)
      at IModelTransformer.onExportElement (lib\cjs\IModelTransformer.js:124:144)
      at IModelExporter.exportElement (src\IModelExporter.ts:161:45)
      at async D:\js\s\imodeljs\core\transformer\src\IModelExporter.ts:148:6
```

Reply:
yes

Reply:
bye.